packages/selectors/src/index.ts

```diff
@@ -0,0 +1,35 @@
+import { createSelector } from "solid-js";
+
+/**
+ * Wrapper around `createSelector` to create a selector for an array.
+ *
+ * @param source - source array signal to create the selector from
+ * @param options - set computation name for debugging purposes
+ * @returns selector for the array
+ *
+ * @see https://github.com/solidjs-community/solid-primitives/tree/main/packages/selectors#createArraySelector
+ *
+ * @example
+ * const list: string[] = [...]
+ * const [selectedItems] = createSignal<string[]>([])
+ * const isSelected = createArraySelector(selectedItems)
+ * <For each={list}>
+ *   {(item) => 
+ *    <li classList={{ active: isSelected(item) }}>
+ *      {item}
+ *    </li>}
+ * </For>
+ */
+export function createArraySelector<T>(
+  source: () => Array<T>,
+  options?: { name?: string },
```

Comment:
Better to reuse types from solid for the options (and any other type if possible), to show where it is coming from, and remove the need for updating the types when options in solid change.